sdk/eventhub/Azure.Messaging.EventHubs/src/Authorization/SharedAccessSignatureCredential.cs

```diff
@@ -84,5 +84,17 @@ public SharedAccessSignatureCredential(SharedAccessSignature signature)
         ///
         public override ValueTask<AccessToken> GetTokenAsync(TokenRequestContext requestContext,
                                                              CancellationToken cancellationToken) => new ValueTask<AccessToken>(GetToken(requestContext, cancellationToken));
+
+        internal void UpdateSharedAccessKey(string keyName, string keyValue)
```

Comment:
nit: I'm guessing that they were left out for the draft iteration and conversation, but we'll want to flesh out the doc comments before we merge in.